api/mod/tests/unit/test_user.py

```diff
@@ -0,0 +1,24 @@
+"""
+Module for unit tests
+"""
+
+
+import hashlib
+
+from mod.src.database import db_models
+
+
+def test_hash_new_password() -> None:
```

Comment:
Also, I'm not completely sure about the value of this test, as it does exactly the same thing the implementation does as well? I'd rather hard-code the hash value, but suppose that's not possible because of the salt?

Reply:
Yeah I think we should just drop that test

Reply:
mock urandom